about/index.md

```diff
@@ -2,9 +2,9 @@
 layout: page
 title: About Me
 tags: [about]
-modified: 2014-08-08T20:53:07.573882-04:00
+modified: 2022-04-29T23:43:27.299-0400
```

Comment:
### [Pebble](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify/#-pebble-non-blocking-but-requires-future-action)

I'm worries that the lack of digits on my [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) timestamp might prevent Jekyll from behaving. Please confirm that it works ahead of merge.